routers/repo/pull.go

```diff
@@ -771,7 +772,9 @@ func TriggerTask(ctx *context.Context) {
 	if ctx.Written() {
 		return
 	}
-	if secret != base.EncodeMD5(owner.Salt) {
+	got := []byte(base.EncodeMD5(owner.Salt))
```

Comment:
Why does this

1. use md5?
2. use owner's salt instead of some secret?

And who calls this API? Couldn't find any callers inside gitea itself.